Homework_Polymor_Exc/Main.java

```diff
@@ -0,0 +1,96 @@
+package Polymorphism.Homework;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
I don't see a code where we are iterating through all shapes and doing calculations on them based on their implementation (triangle, rectangle, circle).

For example, I would expect that user can specify on input:

N number of any shapes (when I say any, I mean triangle/rectangle/circle)
For each shape, user needs to be able to specify inputs so that area/perimeter can be calculated for them
You would store each shape object into the ArrayList
Finally, you would iterate through ArrayList and do calculation for each shape